cfme/fixtures/parallelizer/__init__.py

```diff
@@ -109,7 +109,8 @@ class SlaveDetail(object):
 
     appliance = attr.ib()
     worker_config = attr.ib()
-    id = attr.ib(default=attr.Factory(lambda: next(SlaveDetail.slaveid_generator)))
+    id = attr.ib(default=attr.Factory(lambda: next(SlaveDetail.slaveid_generator)),
+                 repr=lambda value: value.decode('utf-8'))
```

Comment:
@izapolsk can you make this change in the `slaveid_generator`? Something like 

```
slaveid_generator = (f'slave{i:02d}'.encode('utf-8') for i in count())
```

Reply:
@john-dupuy, I can't I'm under impression it is intentionally made `bytes` for interprocess communication and etc

Reply:
Ah gotcha, okay, I was curious